pkg/kclient/kclient.go

```diff
@@ -234,6 +234,21 @@ func (c *Client) IsResourceSupported(apiGroup, apiVersion, resourceName string)
 	return supported, nil
 }
 
+func (c *Client) IsResourceGroupSupported(apiGroup string) (bool, error) {
```

Comment:
Please add a comment for the function

Reply:
I deleted this method. I am no longer using it.